### PR TITLE
Improved turtle mode

### PR DESCRIPTION
**Pull request by direwolfclaw**
### Brief description of the changes:
Turtle mode with variable number of bridges for horizontal and vertical axes.
Names for variables had to be reduced slightly in GUI to accommodate Nx and Ny variables, which are the number of bridges on the horizontal (nhorizontal) and vertical (nvertical) axes.
Default size now large enough to accommodate a reasonably sized main (recognizing the importance of defaults, see Misbehaving by Thaler).
This would be potentially helpful for games with newer and/or smaller numbers of players. 
The additional bridges also help biters pathfind correctly. The single-bridge version has pathfinding issues according to others.
Moats now contain a reasonable but slightly overtuned number of fish. Fish probability could be added as a variable in the GUI, space permitting.

### Tested Changes:
- [x] I've tested the changes locally or with people.
- [ ] I've not tested the changes.

### CODE REVIEW

**Comment by developer-8** [comment]
image
It's a 64x64 moat, it's broken in master too, but in the other direction, maybe you can fix it

File: comfy_panel/special_games.lua [hunk 1]
```diff
@@ -27,15 +28,19 @@ local valid_special_games = {
 	turtle = {
 		name = {type = "label", caption = "Turtle", tooltip = "Generate moat with given dimensions around the spawn"},
 		config = {
-			[1] = {name = "label1", type = "label", caption = "moat width"},
-			[2] = {name = 'moat_width', type = "textfield", text = "5", numeric = true, width = 40},
-			[3] = {name = "label2", type = "label", caption = "entrance width"},
+			[1] = {name = "label1", type = "label", caption = "moat"},
+			[2] = {name = 'moat_width', type = "textfield", text = "10", numeric = true, width = 40},
```
Comment (developer-8) [inline comfy_panel/special_games.lua:32]: You can add `tooltip' to fields to better describe them

File: comfy_panel/special_games.lua [hunk 2]
```diff
@@ -522,13 +560,13 @@ local function on_gui_click(event)
...
 
 	if element.name == "turtle_confirm" then
-
-		local moat_width = config["moat_width"].text
-		local entrance_width = config["entrance_width"].text
-		local size_x = config["size_x"].text
-		local size_y = config["size_y"].text
-
-		generate_turtle(moat_width, entrance_width, size_x, size_y)
+		local moat_width = math.floor(config["moat_width"].text)
```
Comment (developer-8) [inline comfy_panel/special_games.lua:563]: fields are `numeric`, so what is `math.floor` for ? can they be not integers somehow?

**Comment by direwolfclaw** [comment]
I think we're done here

**Pull request closed without merging**